src/core/cli/repl.rs

```diff
@@ -638,8 +639,14 @@ impl<F: PrimeField32, C1: Chipset<F>, C2: Chipset<F>> Repl<F, C1, C2> {
 
     pub(crate) fn run(&mut self) -> Result<()> {
         println!("Lurk REPL welcomes you.");
-
-        let mut editor: Editor<InputValidator<F>, DefaultHistory> = Editor::new()?;
+        let mut config = Config::default();
```

Comment:
Factor the config generation to a dedicated function please. Also, please use constants for those hardcoded strings